Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/E2ETests/E2EAnalysisTests.cs

```diff
@@ -268,7 +263,7 @@ public void E2E_HasManagedAndContentFiles_VB()
         actualStructure.AssertExpectedFileList("\\none1.txt", "\\code1.vb", "\\code2.vb");
     }
 
-    [TestMethod] // SONARMSBRU-104: files under the obj folder should be excluded from analysis
```

Comment:
Is this ticket not relevant anymore?

Reply:
The project no longer exists on Jira. I'm not sure what's the history of this.

Reply:
There are a few more comments like these but those seem to have a more useful message. Maybe I should remove the issue id from all of them and just keep the text.

Reply:
I'm wondering if the message is explaining what is still to be implemented (Jira ticket still todo) or it's explaining the implementation (referencing the closed ticket in Jira). If it's the first, than I'm afraid that leaving the message might be confusing

Reply:
The test already checks that the files from the `obj` folder are excluded (see also the test name). My understanding is that the issue was fixed, and a link was added to the issue for history purposes.

Reply:
Then it's fine for me to remove all the Jira identifiers and leave the comment 👍